csrc/anemone_mempool.h

```diff
@@ -34,6 +37,19 @@ typedef struct {
   void     *maximum_ptr;
 } anemone_mempool_t;
 
+
+inline size_t
+round_up_allocation (size_t num_bytes)
+{
+#if ANEMONE_ALIGNMENT == 16
```

Comment:
I added better comment on the `#define` of that value. 

I could reduce it to:
```
#if ANEMONE_ALIGNMENT == 8
  return ((num_bytes + 7) >> 3) << 3;
#else
#error "Bad ANEMONE_ALIGNMENT value"
#endif
```
because the shifting trick is only valid for `ANEMONE_ALIGNMENT == 8`, but if you didn't know the code, that may not be obvious.